drivers/overlay/ov_serf.go

```diff
@@ -94,8 +94,8 @@ func (d *driver) notifyEvent(event ovNotify) {
 }
 
 func (d *driver) processEvent(u serf.UserEvent) {
-	logrus.Debugf("Received user event name:%s, payload:%s\n", u.Name,
-		string(u.Payload))
+	logrus.Debugf("Received user event name:%s, payload:%s LTime:%v \n", u.Name,
```

Comment:
Yeah we know that. It just looks weird in the same Debufg call you mix the dedicated verb `%s` with the generic one `%v` when you exactly know the type of each argument.

As I said it is minor comment. You don't have to change it.